# User-defined parameters
wam:
  planning:
    DOF: 7
  joint_controller:
    position:
      Kp: [1500, 2500, 600, 500, 50, 50, 50]
      Kd: [50, 20, 5, 2, 0.5, 0.5, 0.05]
      Ki: [25, 10, 4, 1, 5, 5, 5]
      i_lim: [0, 0, 0, 0, 0, 0, 0] 
      u_lim: [25, 20, 15, 15, 5, 5, 5]
    velocity:
      Kp: [42, 42, 18, 18, 3, 3, 3] 
      Kd: [0, 0, 0, 0, 0, 0, 0.1] 
      Ki: [0, 0, 0, 0, 0, 0, 0] 
      i_lim: [0, 0, 0, 0, 0, 0, 0] 
      u_lim: [25, 20, 15, 15, 5, 5, 5]
